descholar-ceo/AnnounceIT
# Ft admin deletes announcement 170727613

#### What does this PR ​ do​ ?
As a user, I should be to navigate in the application so easily

#### Description of Task to be completed?
* Write a javascript function which deletes an element from announcements array
* Add comments to explain that the function will be operational when the real data will be there,
`(real data from the database)`;
* Make User interface easier to the user to navigate easily

#### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

* Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter this to navigate as a normal user `http://127.0.0.1:5500/UI/html/user-profile.html`, and click on one of the announcements appeared on a table,
* and click on the red button written `delete`,
* or `http://127.0.0.1:5500/UI/html/admin-side/admin-page.html` to navigate as an admin and click on `Announcements button or all announcements button`
* You will be redirected to the page of a list of announcements, click on one of them

* Or if you don't use [VSCode](https://code.visualstudio.com/) as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/` and you will see different files, you choose `user-profile.html` and click on one of the announcements appeared on a table,
* and click on the red button written `delete`,
* or go to `AnnounceIT/UI/htm/admin-side/` and choose admin-page.html to navigate as an admin and right click and open with your preferred web browser and click on `Announcements button or all announcements button`
* You will be redirected to the page of a list of announcements, click on one of them

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)

#### What are the relevant pivotal tracker stories?
[170727613](https://www.pivotaltracker.com/n/projects/2429498#)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-21 11-27-06](https://user-images.githubusercontent.com/56886064/72792072-0a34e180-3c41-11ea-82a7-621faf1559e6.png)
![Screenshot from 2020-01-21 11-27-20](https://user-images.githubusercontent.com/56886064/72792073-0acd7800-3c41-11ea-9879-f731828f6e32.png)


#### Questions:
No Question
